src/index.js

```diff
@@ -75,4 +102,28 @@ export default class SaucelabsConnector {
 
         await closeTunnel();
     }
+
+    async waitForFreeMachines (machinesCount, requestInterval, attemptsCount) {
+        var attempt          = 1;

```

Comment:
let's rename it to `attempts` and set to `0` in start. Because in the start we have zero attempts to check free machines.
`attemptsCount` should be `maxAttemptsCount`